build/DependencyVersions.props

```diff
@@ -25,7 +25,7 @@
     <MicrosoftNETSdkPackageVersion>1.0.0-preview-62924-09</MicrosoftNETSdkPackageVersion>
```

Comment:
@dsplaisted - Is the versioning for dotnet/sdk package different? Shouldn't this be 2.1.400-x?

Reply:
We had a bug for a while where the package versions went back to 1.0.0.  The latest packages out of the 4xx branch should have a 2.1.400 version.